### ISSUE TITLE
Can Pylians compute the power spectrum of the vorticity for a 3D velocity field?

### ISSUE THREAD

**Issue by WeakDog** [franciscovillaescusa/Pylians3#19]
I’m interested in whether Pylians3 can compute the power spectrum of the vorticity for a 3D velocity field. I noticed that the documentation covers computing the power spectrum of the velocity divergence, but I couldn’t find any information on calculating the vorticity power spectrum. Is it possible to compute this with the current version of Pylians3?

Thanks in advance for any guidance!

**Comment by franciscovillaescusa** [comment]
Hi,

Pylians doesn't have a function to compute the vorticity power spectrum right now, but I think it would be pretty easy to implement. Let me know if you want to talk about this.

Best

Francisco

**Comment by WeakDog** [comment]
Yes, I would greatly appreciate it if there is an example demonstrating how to calculate the vorticity power spectrum in Pylians.
